user/registration/src/main/java/in/projecteka/jataayu/registration/ui/fragment/RegistrationOtpFragment.kt

```diff
@@ -40,30 +41,32 @@ class RegistrationOtpFragment : BaseFragment() {
         return binding.root
     }
 
-    override fun onVisible() {
-        super.onVisible()
-    }
-
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         binding.viewModel = viewModel
         parentVM.appBarTitle.set(getString(R.string.verification))
 
-        viewModel.otpMessageLbl.set(SpannableStringBuilder()
-            .append(getString(R.string.otp_sent))
-            .bold { append(" ${parentVM.getIdentifierValue()}") })
-
```

Comment:
Confirm why the identifier value has been removed.

Reply:
I have added it in the next line. So there was already a TextView for identifier which was not being used. That's why we removed it from the `otpMessageLbl` and added it to `mobileNumberText`.